packages/atlas/src/components/_nft/NftWidget/NftWidget.tsx

```diff
@@ -130,6 +131,7 @@ export const NftWidget: FC<NftWidgetProps> = ({
         <NftInfoItem
           size={size}
           label="Buy now"
+          disableSecondary={!JOY_PRICE_SERVICE_URL}
```

Comment:
Same here, let's not rely on the URL and instead just do `convertHapiToUSD(buyNowPrice)` in a variable and check if that's a truthy value